quic/s2n-quic-core/src/recovery/bbr/windowed_filter.rs

```diff
@@ -87,6 +81,89 @@ impl<
     pub fn value(&self) -> Option<T> {
         self.current_value
     }
+
+    #[inline]
+    fn window_expired(&self, now: TimeType) -> bool {
+        self.last_updated.map_or(false, |last_updated| {
+            now - last_updated >= self.window_length
+        })
+    }
+}
+//= https://tools.ietf.org/id/draft-cardwell-iccrg-bbr-congestion-control-02#4.5.3.2
+//# A BBR implementation MAY use a generic windowed min filter to track BBR.min_rtt.
+//# However, a significant savings in space and improvement in freshness can be achieved
+//# by integrating the BBR.min_rtt estimation into the ProbeRTT state machine
+
+/// Specialized WindowedMinFilter for tracking min round trip time
+///
+/// BBRv2 tracks both a min probe RTT that is refreshed at least every `PROBE_RTT_INTERVAL`,
+/// and an overall min_rtt that is refreshed at least every `MIN_RTT_FILTER_LEN`.
+#[derive(Clone, Debug)]
+pub(crate) struct MinRttWindowedFilter {
+    min_probe_rtt: WindowedFilter<Duration, Timestamp, Duration, MinFilter>,
+    min_rtt: WindowedFilter<Duration, Timestamp, Duration, MinFilter>,
+    //= https://tools.ietf.org/id/draft-cardwell-iccrg-bbr-congestion-control-02#2.14.2
+    //# A boolean recording whether the BBR.probe_rtt_min_delay has expired and is due for a
+    //# refresh with an application idle period or a transition into ProbeRTT state.
+    probe_rtt_expired: bool,
+}
+
+//= https://tools.ietf.org/id/draft-cardwell-iccrg-bbr-congestion-control-02#2.14.2
+//# A constant specifying the minimum time interval between ProbeRTT states: 5 secs.
+const PROBE_RTT_INTERVAL: Duration = Duration::from_secs(5);
+
+//= https://tools.ietf.org/id/draft-cardwell-iccrg-bbr-congestion-control-02#2.14.1
+//# A constant specifying the length of the BBR.min_rtt min filter window,
+//# MinRTTFilterLen is 10 secs.
+const MIN_RTT_FILTER_LEN: Duration = Duration::from_secs(10);
```

Comment:
```suggestion
const MIN_RTT_FILTER_INTERVAL: Duration = Duration::from_secs(10);
```

Reply:
I'd prefer to keep the name from the RFC